app/presenters/sufia/file_set_presenter.rb

```diff
@@ -4,6 +4,14 @@ class FileSetPresenter < ::CurationConcerns::FileSetPresenter
 
     delegate :depositor, :tag, :date_created, :date_modified, to: :solr_document
 
+    def collector
```

Comment:
Can you remove all references to `collector` in this PR? It's a role that exists in CC but hasn't existed in Sufia, so I'd like to revisit roles more broadly in a future PR.